src/modules/client/pages/ni/billing/ClientsBalances.vue

```diff
@@ -125,6 +125,15 @@ export default {
         { name: 'billed', label: 'Facturé TTC', align: 'center', field: 'billed', format: val => formatPrice(val) },
         { name: 'paid', label: 'Payé TTC', align: 'center', field: 'paid', format: val => formatPrice(val) },
         { name: 'balance', label: 'Solde', align: 'center', field: row => row.balance, sortable: true },
+        {
+          name: 'lastCesuPaymentDate',
+          label: 'Date du dernier prelevement CESU',
+          field: 'lastCesuPaymentDate',
+          align: 'center',
+          sortable: true,
+          format: value => this.formatDate(value),
```

Comment:
je crois que tu peux simplement mettre `format: this.formatDate`
et du coup simplifier les autres aussi sous ce format

Reply:
done